src/django-app/django-project/shops/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+    
+from django.utils.translation import gettext as _
+
+class Shop(models.Model):
+    shop_owner = models.ForeignKey("accounts.ShopOwner", on_delete=models.CASCADE)
+    address = models.OneToOneField("Address", on_delete=models.PROTECT)
+    name = models.CharField(_("shop name"), max_length=255)
+    num_employees = models.IntegerField(_("number of employees"), default=0, null=True, blank=True)
+    num_bays = models.IntegerField(_("number of service bays"), default=1, null=True, blank=True)
+    availability = models.CharField(_("available hours"), max_length=255, null=True, blank=True)
+    shop_hours = models.CharField(_("shop hours"), max_length=255, null=True, blank=True)
+
+    class Meta:
+        verbose_name = "Shop"
+        verbose_name_plural = "Shops"
+
+class Address(models.Model):
```

Comment:
Should `Address` be in its own app? 🤔 It doesn't really matter but we will be using the same Address model in other parts of the app that are not related to the shop.

Reply:
Yeah, I was thinking about that too. I think it'd be a bit weird for it to be it's own app, that's why I just threw it into shops, since the only other parts of the app that use it are employees. I could move it, but it seems a bit odd to me to have an app *just* for addresses.

Reply:
Yeah, I agree. You can leave it in `shops` app for now (or move it to `accounts` if you'd like). We might need to make a new app in the future that contains shared models among the users and we can move `Address` to that if needed.

Reply:
Sounds good. I'll leave it there for now.